### DISCUSSION TITLE
Adding pronouns property

### DISCUSSION THREAD

**Discussion by shaedrich** [schemaorg/schemaorg#2925] [Q&A] [answered]
Hi, I don't know if you already considerd that but besides the [`gender` property](https://schema.org/gender) (which you solved well by defining it as `GenderType|Text`) there's an increasing use of `pronouns`  on the web so what do you think of adding it as a property?

**Comment by thadguidry** [comment, marked as answer]
I think this is probably a "good thing" to have for structured data about [Person](https://schema.org/Person)
I'd say that a simple property on that called `pronoun` or `genderPronouns` might work?
And it could have an Expected Type of `GenderPronounType` or `Text`?

Is that what you were thinking?

**Reply by shaedrich** [reply]
Yes, that's exactly what I'm thinking.

**Reply by zichy** [reply]
I think it should be `pronouns` since there could be multiple pronouns.

**Reply by shaedrich** [reply]
Good point! People often put "he/him", "she/her", "they/their", etc. in their bios. While this could be a string, allowing for multiple values probably makes sense 🤔 I'm not an expert, but wouldn't this (at least in English, a tuple of length 3: $('he', 'him', 'his')$?

**Reply by KalleOlaviNiemitalo** [reply]
I expect a string "he/him" would work better than an ordered list ("he", "him", "his").

- A string can be shown to users as is, but a more complex structure would have to be transformed first.  The more complex structure might in principle be better for software that reads this property and generates gender-specific text accordingly; but if it is too complex, then users will not input the data correctly.
- Multiple values are not automatically ordered.  If the order matters, then it has to be set explicitly, like <https://schema.org/position> in <https://schema.org/ListItem>, further complicating the structure.
- Might be difficult to specify the expected order of case forms in such a way that it makes sense for English as well as other languages.
- Multiple values might be better used for language-tagged strings as in JSON-LD [value objects](https://www.w3.org/TR/json-ld11/#value-objects) so you can specify the pronouns in English as one value and the pronouns in Russian as another value, for example.

**Reply by shaedrich** [reply]
The `gender` property offers both `GenderType` and `Text`&mdash;so that would be an option. With `PronounType` there would be the option to have a `language` property on that type but making the type's own `gender` still an enumeration might have the problems, you described. I'm not sure, what schema.org generally thinks of having two properties, one literal and one semantic like `GenderGrammar` so that applications only ever use one, depending on if they parse the value or use it as is, but that might be too complex.

**Reply by eyaler** [reply]
Please note that non-English languages may have gendered pronouns also for second or even first person, which may require multiple lists, and/or assign different semantic meaning to items of a single list. Perhaps this should be a key value where key is the person and value is the list of subject/object/possesive. e.g. for Hebrew:
{'3rd': ['הוא', 'אותו', 'שלו'], '2nd': ['אתה', 'אותךָ', 'שלך']}

**Reply by shaedrich** [reply]
Good catch! Thanks for the hint! :)

**Reply by shaedrich** [reply]
> Good point! People often put "he/him", "she/her", "they/their", etc. in their bios. While this could be a string, allowing for multiple values probably makes sense 🤔 I'm not an expert, but wouldn't this (at least in English, a tuple of length 3: $('he', 'him', 'his')$?

https://pronouny.xyz/ even lists the tuple $('they', 'them', 'their', 'theirs', 'theirself')$ (interestingly, not "themself" and "themselves" 🤔 )

**Reply by MatthiasWiesmann** [reply]
I second [danbri](https://github.com/danbri)'s position we should leverage an existing standard: different languages handle pronouns differently and require a different number of values for different grammatical constructs / cases – for instance you would probably need 7 for Serbian. And you would need a consensus on how to name the keys.

**Reply by shaedrich** [reply]
I'm open for suggestions. The most important thing right now is that this doesn't go stale over and over again.

**Reply by MatthiasWiesmann** [reply]
I would suggest making the pronoun an instance of https://schema.org/inDefinedTermSet with one term being one form of the pronoun, with a [termCode](https://schema.org/termCode) matching the grammatical case.

```json
"pronoun": {
  "@type": "DefinedTermSet",
  "hasDefinedTerm": {
     "@type": "DefinedTerm",
     "name": "Xe",
     "termCode": "Nominative"
  },
   "hasDefinedTerm": {
     "@type": "DefinedTerm",
     "name": "Xem",
     "termCode": "Accusative"
  },
  "hasDefinedTerm": {
     "@type": "DefinedTerm",
     "name": "Xir",
     "termCode": "Possesive Adjective"
  },
 "hasDefinedTerm": {
     "@type": "DefinedTerm",
     "name": "Xirs",
     "termCode": "Genitive"
  },
 }
```

**Reply by shaedrich** [reply]
Interesting. We didn't have that case yet. I'm curious what the others think about that solution 👍🏻

**Comment by danbri** [comment]
It would be very preferable to cite another standard for this, if there is
something suitable out there.

Even in English, if I write “they” for pronouns, am I to be understood as
telling you something about my gender, or about how I want sentences about
me to written and said. Personally I wish everyone used “they” for
describing me, to encourage a climate in which people are mentioned without
their gender always being declared, much as the adoption of “Ms” achieved
for marital status some years ago. However that is goal that may not make
sense in other languages, and is likely not what most people have in mind
when filling out this field.

Some years ago, the old social network Orkut had a “looking for ( men /
women / men and women )” field. Orkut had unexpectedly taken off in Iran
and it turned out that many users there simply missed the
dating/relationships/sex subtext which that profile field had in its
original context, resulting in high % listing “both” (it seems reading this
as an expression of business networking and friend-making interest, or not
reading it at all). This was in a country where homosexuality can result in
people being murdered by the state (
https://en.wikipedia.org/wiki/LGBT_rights_in_Iran).

What I mean by this is that it can be problematic to assume that the ‘real’
intent of a field is universally understood, over and above the definition
given it in schemas and UI. For pronouns, there is so much that is unsaid…
yet understood by those who have learned about it elsewhere.

**Reply by shaedrich** [reply]
> Even in English, if I write “they” for pronouns, am I to be understood as
telling you something about my gender, or about how I want sentences about
me to written and said.

So, what, in your opinion, would be a more suiting key for this property?

> What I mean by this is that it can be problematic to assume that the ‘real’
intent of a field is universally understood, over and above the definition
given it in schemas and UI.

I think, the primary task for structured, semantic metadata properties is to be neutral, clear, and follow [POLS](https://de.wikipedia.org/wiki/Principle_of_Least_Surprise). Language, and grammar in particular—with exceptions of course—is structured in such a way, that we can borrow from it and reach an understanding in that particular language since people usually know a fair amount about their language's grammar (assuming, they are fluent in it).

> Personally I wish everyone used “they” for
describing me, to encourage a climate in which people are mentioned without
their gender always being declared, much as the adoption of “Ms” achieved
for marital status some years ago.

When people say "Why labels?" (be it a pronoun, gender, orientation, whatever), I agree, that a world without labels would be ideal, yet, we cannot achieve that by just attempting to abandon labels. That would only cement the status quo ("*-normativity" including the generic masculine). I think of labels as a "transitional characteristic" of society's progressivness towards a more open and equal co-existence. Therefore, we need to be aware of said status quo to change the way, we perceive it.

Because, what people—primarily but not exclusively trans, inter and non-binary people—face, is that their gender is just assumed. On the other hand, forcing people to completely ignore gender might sound like something that might make the world a more equal place but actually results again in repression. Self-identity is something very personal and we should let people deal with it. Also, a luxury (not to say "privilege"), trans, inter and non-binary people don't have compared to non-heterosexual people, is that they can't just keep that topic "to their bedrooms" because it will be brought up in daily life one way or the other and be it in [microaggressions](https://en.wikipedia.org/wiki/Microaggression).

Another thing is, people assume, pronouns are an exclusively non-cis topic but that is everything but inclusive thinking. While people might have the need to clarify their pronouns, they don't necessarily also want to come out as trans, inter and non-binary, which they implicitly would have to if only they would be using pronouns. That's why only if cis people also use pronouns explicitly, forced outing can be avoided.

> Some years ago, the old social network Orkut had a “looking for ( men /
women / men and women )” field. Orkut had unexpectedly taken off in Iran
and it turned out that many users there simply missed the
dating/relationships/sex subtext which that profile field had in its
original context

The funny thing is, to make this sexual/romantic orientation thing work, it's usually not absolute but relative to one's own gender if one isn't polysexual/-romantic
